### PR TITLE
DATAUP-497: minor fixes

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

A few minor fixes spotted while working on DATAUP-497

Removing / fixing invalid CSS
Remove code for condition that will never occur
Small app/BI cell title restyle

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-497
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and navigating to see that nothing has changed (but if you look at the CSS warnings in the console, there should not be any from all_concat.css)

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/scss/partials/_appCell.scss [hunk 1]
```diff
@@ -141,7 +141,6 @@
 
   .kb-app-footer {
     background-color: rgb(245 245 245);
-    overflow: none;
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_appCell.scss:144]: invalid value

File: kbase-extension/scss/partials/_cellToolbar.scss [hunk 2]
```diff
@@ -29,6 +29,7 @@ $default-icon-size: 56px;
   &__title-container {
     flex: 1 1 10rem;
     text-overflow: ellipsis;
+    overflow: hidden;
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_cellToolbar.scss:32]: hide overflowing text in the app title and subtitle

File: kbase-extension/scss/partials/_narrative.scss [hunk 3]
```diff
@@ -568,7 +568,7 @@ p#site-title { /* FIXME */
 
 #data-add-btn {
   font-size: 0.8em;
-  padding: 0 0 -10px;
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_narrative.scss:571]: invalid

File: kbase-extension/scss/partials/_landingPages.scss [hunk 4]
```diff
@@ -71,7 +71,7 @@ table {
 
 .scroll-pane {
   height: 200px;
-  overflow-x: hi;
```
Comment (ialarmedalien) [inline kbase-extension/scss/partials/_landingPages.scss:74]: hi overflow!

File: test/unit/testUtil.js [hunk 5]
```diff
@@ -55,8 +55,8 @@ define('testUtil', ['bluebird', 'util/stagingFileCache', 'json!/test/testConfig.
         });
     }
 
-    function JSONcopy(structure) {
-        return JSON.parse(JSON.stringify(structure));
+    function JSONcopy(...args) {
+        return JSON.parse(JSON.stringify(...args));
```
Comment (ialarmedalien) [inline test/unit/testUtil.js:59]: allow extra args to JSON.stringify

File: test/unit/spec/nbextensions/appCell2/widgets/tabs/resultsViewer-spec.js [hunk 6]
```diff
@@ -161,24 +161,6 @@ define([
             viewerNode.remove();
         });
 
-        it('starts and stops a viewer with a report from a non-batch job', async () => {
```
Comment (ialarmedalien) [inline test/unit/spec/nbextensions/appCell2/widgets/tabs/resultsViewer-spec.js:164]: removing test for the data structure that never occurs

File: kbase-extension/scss/partials/_landingPages.scss (hunk 4 above)
Comment (briehl) [reply kbase-extension/scss/partials/_landingPages.scss:74]: 👋

**Review by briehl — APPROVED** [review]

**Pull request merged**